terraform-environments/iam-role.tf

```diff
@@ -0,0 +1,90 @@
+#Definition of IAM users and groups
+
+resource "aws_iam_user" "devops" {

```

Comment:
why to use same resource 3 times and create 3 users, instead use count or some other logic and use the resource once.